ddspipe_participants/src/cpp/writer/auxiliar/RtpsEchoWriter.cpp

```diff
@@ -60,6 +60,7 @@ utils::ReturnCode RtpsEchoWriter::write(
                 " in topic: " << topic_.topic_name() <<
                 " payload received: " << rtps_data.payload <<
                 " with specific qos: " << rtps_data.writer_qos <<
+                " with original source: " << rtps_data.original_source_guid <<
```

Comment:
Could you please also add other useful information such as source_timestamp or source_guid? Idea for the future: let the user customize which information to show through yaml configuration, instead of having everything (verbose mode) or almost nothing.